pkg/api/aim2/service/run/converters.go

```diff
@@ -0,0 +1,30 @@
+package run
+
+import (
+	"encoding/json"
+
+	"github.com/rotisserie/eris"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim2/api/request"
+	"github.com/G-Research/fasttrackml/pkg/api/aim2/dao/dto"
+)
+
+// ConvertRunMetricsRequestToMetricKeysMapDTO converts request of `GET /runs/:id/metric/get-batch`
+// endpoint to internal DTO object.
+func ConvertRunMetricsRequestToMetricKeysMapDTO(req *request.GetRunMetricsRequest) (dto.MetricKeysMapDTO, error) {
```

Comment:
this function name is kind a mouthful -- i think the param and return types don't really need to be in the name since they are in the signature